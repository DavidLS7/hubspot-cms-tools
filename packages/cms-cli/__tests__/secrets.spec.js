/*global banjo */

describe('Secrets command', () => {
  it('works with help positional', async () =>
    expect(await banjo(['secrets', 'help'])).toMatchSnapshot());

  it('works with help option', async () =>
    expect(await banjo(['secrets', '--help'])).toMatchSnapshot());
});
