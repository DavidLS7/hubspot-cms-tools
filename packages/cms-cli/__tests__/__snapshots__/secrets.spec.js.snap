// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Secrets command works with help option 1`] = `
"banjo secrets

Manage HubSpot secrets

Commands:
  banjo secrets list                   List all HubSpot secrets
  banjo secrets add <name> <value>     Add a HubSpot secret
  banjo secrets update <name> <value>  Update an existing HubSpot secret
  banjo secrets delete <name>          Delete a HubSpot secret

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
"
`;

exports[`Secrets command works with help positional 1`] = `
"banjo secrets

Manage HubSpot secrets

Commands:
  banjo secrets list                   List all HubSpot secrets
  banjo secrets add <name> <value>     Add a HubSpot secret
  banjo secrets update <name> <value>  Update an existing HubSpot secret
  banjo secrets delete <name>          Delete a HubSpot secret

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
"
`;
